include/mbedtls/x509_crt.h

```diff
@@ -146,7 +146,8 @@ typedef struct mbedtls_x509_subject_alternative_name {
     int type;                              /**< The SAN type, value of MBEDTLS_X509_SAN_XXX. */
     union {
         mbedtls_x509_san_other_name other_name; /**< The otherName supported type. */
-        mbedtls_x509_buf   unstructured_name; /**< The buffer for the un constructed types. Only dnsName currently supported */
+        mbedtls_x509_name directory_name;
```

Comment:
Please add description for this field for consistency.

Reply:
I'm not sure if this is beneficial. I would even remove the otherName description, as it doesn't introduce any new information :D